### PR TITLE
Add InterAP comm for voice detect

### PR DESCRIPTION
**Pull request by chaitanyagodavarthi**

### CODE REVIEW

File: feeds/wlan-ap/interAPcomm/Makefile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  SECTION:=base
+  CATEGORY:=Base system
+  TITLE:=Inter AP communication library
+endef
+
+define Package/libinterapcomm/description
+ Inter AP communication library.
+endef
+
+define Build/Prepare
```
Comment (blogic) [inline feeds/wlan-ap/interAPcomm/Makefile:22]: you can drop the Build/Prepare section as it duplicates the functionality that is default behaviour

File: feeds/wlan-ap/interAPcomm/Makefile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+PKG_NAME:=libinterapcomm
+PKG_RELEASE:=1.0.0
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libinterapcomm
+  SECTION:=base
```
Comment (blogic) [inline feeds/wlan-ap/interAPcomm/Makefile:13]: these 2 fields appear to be missing
SECTION:=libs
CATEGORY:=Libraries

**Comment by blogic** [comment]
the patch has a formal error ... the patch header should be

package: new feature

description of new feature

Signed-off-by: Name <email addr>

File: feeds/wlan-ap/interAPcomm/src/src/interAPcomm.c [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+	/* Set socket to allow broadcast */
+	bcast_perm = 1;
+	if (setsockopt(recv_sock, SOL_SOCKET, SO_BROADCAST, (void *) &bcast_perm,
+			sizeof(bcast_perm)) < 0)
+		printf("setsockopt failed");
+
+	memset(&addr, 0, sizeof(addr));
+	addr.sin_family = AF_INET;
+	addr.sin_addr.s_addr = htonl(INADDR_ANY);  /* Any incoming interface */
+//	addr.sin_addr.s_addr = htonl(INADDR_BROADCAST);
```
Comment (yashvardhan-netex) [inline feeds/wlan-ap/interAPcomm/src/src/interAPcomm.c:66]: Please remove commented code from all the files

File: feeds/wlan-ap/interAPcomm/src/src/test/Makefile [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+OBJS = interapcommtest.o
+
+CFLAGS += -Wall $(EXTRA_CFLAGS) $(extra_cflags-y)
+
+LIBS =	-L ../../ -linterapcomm
+
+CFLAGS += -I. \
+	-I../../include/
+
+LDFLAGS := $(subst EB,mbig-endian,$(LDFLAGS))
```
Comment (yashvardhan-netex) [inline feeds/wlan-ap/interAPcomm/src/src/test/Makefile:11]: Do we really need this?

File: feeds/wlan-ap/modules/nf_uccdetect/src/nl_ucc.h [hunk 5]
```diff
@@ -35,6 +35,31 @@ struct capture_buf
...
+    uint32_t RtpIpTo;
+};
+
+#if 0
+struct ex {
+	int x;
+	char y;
+	int z;
+};
+#endif
```
Comment (yashvardhan-netex) [inline feeds/wlan-ap/modules/nf_uccdetect/src/nl_ucc.h:61]: Do we need it in future? If not, Can we remove it?

File: feeds/wlan-ap/interAPcomm/Makefile (hunk 2 above)
Comment (blogic) [reply feeds/wlan-ap/interAPcomm/Makefile:13]: It appears that the comments were not addressed in the latest version of the patch

File: feeds/wlan-ap/interAPcomm/Makefile (hunk 1 above)
Comment (blogic) [reply feeds/wlan-ap/interAPcomm/Makefile:22]: It appears that the comments were not addressed in the latest version of the patch

File: feeds/wlan-ap/interAPcomm/src/src/interAPcomm.c (hunk 3 above)
Comment (blogic) [reply feeds/wlan-ap/interAPcomm/src/src/interAPcomm.c:60]: It appears that the comments were not addressed in the latest version of the patch

File: feeds/wlan-ap/interAPcomm/src/src/test/Makefile (hunk 4 above)
Comment (chaitanyagodavarthi) [reply feeds/wlan-ap/interAPcomm/src/src/test/Makefile:11]: Yes

**Comment by blogic** [comment]
merged into pending, Thanks !

**Comment by blogic** [comment]
this is causing build failures

**Comment by blogic** [comment]
merged into pending, Thanks !

**Pull request closed without merging**